enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/IdentityVerificationController.java

```diff
@@ -370,41 +380,27 @@ public ObjectResponse<DocumentVerificationSdkInitResponse> initVerificationSdk(
      * @throws IdentityVerificationException Thrown when identity verification is invalid.
-     * @throws PresenceCheckLimitException Thrown when presence check limit is exceeded.
-     * @throws RemoteCommunicationException Thrown when communication with PowerAuth server fails.
-     * @throws OnboardingProcessLimitException Thrown when maximum failed attempts for identity verification have been reached.
      */
     @PostMapping("presence-check/init")
     @PowerAuthEncryption(scope = EciesScope.ACTIVATION_SCOPE)
     @PowerAuth(resourceId = "/api/identity/presence-check/init", signatureType = {
             PowerAuthSignatureTypes.POSSESSION
     })
-    public ObjectResponse<PresenceCheckInitResponse> initPresenceCheck(@EncryptedRequestBody ObjectRequest<PresenceCheckInitRequest> request,
-                                                                       @Parameter(hidden = true) EciesEncryptionContext eciesContext,
-                                                                       @Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication)
-            throws PowerAuthAuthenticationException, DocumentVerificationException, PresenceCheckException,
-            PresenceCheckNotEnabledException, PowerAuthEncryptionException, OnboardingProcessException, PresenceCheckLimitException, RemoteCommunicationException, IdentityVerificationException, OnboardingProcessLimitException {
+    public ResponseEntity<Response> initPresenceCheck(@EncryptedRequestBody ObjectRequest<PresenceCheckInitRequest> request,
+                                                      @Parameter(hidden = true) EciesEncryptionContext eciesContext,
+                                                      @Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication)
+            throws IdentityVerificationException, PowerAuthAuthenticationException, PowerAuthEncryptionException {
 
         checkApiAuthentication(apiAuthentication, "initializing presence check");
         checkEciesContext(eciesContext, "initializing presence check");
         checkRequestObject(request, "initializing presence check");
 
-        if (!identityVerificationConfig.isPresenceCheckEnabled()) {
-            throw new PresenceCheckNotEnabledException();
-        }
-
         final OwnerId ownerId = PowerAuthUtil.getOwnerId(apiAuthentication);
         final String processId = request.getRequestObject().getProcessId();
 
-        onboardingService.verifyProcessId(ownerId, processId);
-
-        final SessionInfo sessionInfo = presenceCheckService.init(ownerId, processId);
-
-        final PresenceCheckInitResponse response = new PresenceCheckInitResponse();
-        response.setSessionAttributes(sessionInfo.getSessionAttributes());
-        return new ObjectResponse<>(response);
+        var stateMachine = stateMachineService.processStateMachineEvent(ownerId, processId, OnboardingEvent.PRESENCE_CHECK_INIT);
```

Comment:
Usage of `var` is a bit inconsistent with the rest of the code. Let's keep the old approach until we switch to `var` globally.